pkgs/misc/fastly/default.nix

```diff
@@ -37,7 +37,107 @@ buildGoModule rec {
     "-X github.com/fastly/cli/pkg/revision.GoHostOS=${go.GOHOSTOS}"
     "-X github.com/fastly/cli/pkg/revision.GoHostArch=${go.GOHOSTARCH}"
   ];
-  preBuild = ''
+  preBuild = let
+    # This file is retrieved from
+    # https://developer.fastly.com/api/internal/cli-config. In order
```

Comment:
Please fetch https://web.archive.org/web/20221104122906/https://developer.fastly.com/api/internal/cli-config instead of vendoring this.

Reply:
For some reason, the downloaded file with `fetchurl` has markup:

```
❯ result/bin/fastly

ERROR: the configuration file is invalid: (1, 1): parsing error: keys cannot contain < character.
```

If I try to download the same URL (https://web.archive.org/web/20221104122906/https://developer.fastly.com/api/internal/cli-config) with curl or wget, I get the raw text file without markdown.

Diff on top of this PR: https://gist.github.com/ereslibre/74b5abaef9e6b49df65fc4f8d8c51f5e

Reply:
PEBCAK.